tools/test/scripts/doxy-spellchecker/spell.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
```

Comment:
Add at least SPDX specifier here (license missing)

Reply:
@0xc0170 If this needs a SPDX header, then does .travix.yml as well?